openquake/commonlib/oqvalidation.py

```diff
@@ -230,6 +230,11 @@ def __init__(self, **names_vals):
             raise ValueError('asset_correlation != {0, 1} is no longer'
                              ' supported')
 
+        # check for GMFs from file
+        if (self.inputs.get('gmfs', '').endswith('.csv') and not self.sites and
+                'sites' not in self.inputs):
+            raise ValueError('You forgot sites|sites_csv in the job.ini!')
```

Comment:
Job configuration file to be more general than `job.ini`?

Reply:
I want to avoid confusions with the openquake.cfg configuration file. Perhaps I should just say `job .ini file`.